test/ik/test_differential_ik.py

```diff
@@ -0,0 +1,66 @@
+from os.path import dirname, join, abspath
+import numpy as np
+import pinocchio
+
+from pyroboplan.core.utils import get_random_transform
+from pyroboplan.ik.differential_ik import DifferentialIk, DifferentialIkOptions
+
+
+def load_panda_models():
+    pinocchio_model_dir = join(dirname(str(abspath(__file__))), "..", "..", "models")
+    package_dir = join(pinocchio_model_dir, "panda_description")
+    urdf_filename = join(package_dir, "urdf", "panda.urdf")
+    return pinocchio.buildModelsFromUrdf(
+        urdf_filename, package_dirs=pinocchio_model_dir
+    )
+
+
+def test_ik_solve_trivial_ik():
+    model, _, _ = load_panda_models()
+    data = model.createData()
+    target_frame = "panda_hand"
+
+    # Initial joint states
+    q = np.array([0.0, 1.57, 0.0, 0.0, 1.57, 1.57, 0.0, 0.0, 0.0])
+
+    # Set the target transform to the current joint state's FK
+    target_frame_id = model.getFrameId(target_frame)
+    pinocchio.framesForwardKinematics(model, data, q)
+    target_xform = data.oMf[target_frame_id]
```

Comment:
Nit: I've been standardizing on using `tform` instead of `xform`, if you don't mind find/replacing.

Reply:
Noted.